open-metadata-implementation/access-services/README.md

```diff
@@ -84,6 +84,10 @@ that support the business.
 models for an organization.  It is designed for architecture tools.  It is able to support the
 definition and management of a digital service through concept to deployment.
 
+* **[digital-service](dev-ops)** - manage metadata for a the Egeria Digital Services lifecycle.
```

Comment:
link to digital-service rather than dev-ops

Reply:
resolved